src/BaseDraw.php

```diff
@@ -760,7 +760,7 @@ public function getLegendSize(array $Format = [])
                     $vY = $vY + max($this->FontSize * count($Lines), $IconAreaHeight) + 5;
                 } elseif ($Mode == LEGEND_HORIZONTAL) {
                     $Lines = preg_split("/\n/", $Serie["Description"]);
-                    $Width = "";
+                    $Width = [];
```

Comment:
Alternatively we could replace this with `$MaxWidth = 0;`  and replace `$Width[] = $BoxArray[1]["X"];` with `$MaxWidth = max($MaxWidth, $BoxArray[1]["X"]);`